RELEASES.md

```diff
@@ -0,0 +1,49 @@
+# Releases
+
+This document describes how Strimzi handles versioning, releases new versions and communicates about them.
+
+Strimzi uses [semantic versioning](https://semver.org/) as a versioning schema for the releases of all the components.
+
+## Release Cycle
+
+Strimzi is using a release schedule of 1 or 2 months for the "core" component, the cluster operator.
+The [GitHub milestones](https://github.com/strimzi/strimzi-kafka-operator/milestones) are used to track the features and bug fixes included in each release.
+
+The other components, under the Strimzi umbrella, don't have a specific release cycle but when having a new releases is decided on a case-by-case basis depending on the new features added, bugs and vulnerabilities fixed.
+
+## Release Process
+
+Most of the Strimzi components have a corresponding documentation about how to make a new release.
+For example, the release process for the cluster operator is described [here](https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/RELEASE.md).
```

Comment:
Should we talk here about release candidates? Where they are announced, when are they done, how long at minimum will be given for testing etc.?

Reply:
Taking a look at other projects, i.e. KEDA, I didn't see RCs mentioned there but I have no strong opinion about not mentioning them. The announcement follow the same process and I guess 2-3 days are enough for testing?

Reply:
I have written something, let's see if that's ok.